### ISSUE TITLE
recipe infra failure: Uncaught Exception: ValueError('Expecting property name: line 139 column 7 (char 3846)',)

### ISSUE THREAD

**Issue by HansMuller** [flutter/flutter#69827]
For https://github.com/flutter/flutter/issues/69827, tests are failing due to "recipe infra failure", for example: https://ci.chromium.org/p/flutter/builders/try/Linux%20analyze/5402?

Swan song says: 
```
recipe infra failure: Uncaught Exception: ValueError('Expecting property name: line 139 column 7 (char 3846)',)
```

**Comment by jmagman** [comment]
Fixed by #69828, rebase and try again.  Sorry about that!

**Comment by HansMuller** [comment]
No problem, thanks for the update!
